clients/src/test/java/org/apache/kafka/common/protocol/types/ProtocolSerializationTest.java

```diff
@@ -39,14 +39,16 @@ public void setup() {
                                  new Field("int64", Type.INT64),
                                  new Field("string", Type.STRING),
                                  new Field("bytes", Type.BYTES),
+                                 new Field("nullable_bytes", Type.NULLABLE_BYTES),
                                  new Field("array", new ArrayOf(Type.INT32)),
-                                 new Field("struct", new Schema(new Field("field", Type.INT32))));
+                                 new Field("struct", new Schema(new Field("field", new ArrayOf(Type.INT32)))));
         this.struct = new Struct(this.schema).set("int8", (byte) 1)
                                              .set("int16", (short) 1)
                                              .set("int32", 1)
                                              .set("int64", 1L)
                                              .set("string", "1")
-                                             .set("bytes", "1".getBytes())
```

Comment:
Same as above. The old test didn't catch the bug because it just expected a SchemaException to be raised.